packages/client/src/Loc.ts

```diff
@@ -1184,27 +1206,60 @@ export class AcceptedRequest extends ReviewedRequest {
         return await this.refresh() as OpenLoc
     }
 
+    private checkOpenParams(): OpenPolkadotLocParams {
+        const requesterAddress = this.request.requesterAddress;
+        if (requesterAddress === undefined || requesterAddress?.type !== "Polkadot") {
+            throw Error("Only Polkadot requester can open LOC");
```

Comment:
Maybe this is equivalent but the `new` keyword is missing.

Reply:
See my answer below.